### PR TITLE
Clarifying TAP-5 is optional but supported

### PR DESCRIPTION
**Pull request by JustinCappos**

### CODE REVIEW

File: uptane-standard.md [hunk 1]
```diff
@@ -835,7 +835,7 @@ Full verification of metadata means that the ECU checks that the Targets metadat
 
 Full verification MAY be performed by either primary or secondary ECUs. The procedure is the same, except that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
 
-A primary ECU SHALL download metadata and images following the rules specified in {{TAP-5}}, and the metadata file renaming rules specified in {{metadata_filename_rules}}.
+A primary ECU SHALL download metadata and images following the rules specified in {{TAP-5}} (if supported) or using the {{TUF-spec}}, and the metadata file renaming rules specified in {{metadata_filename_rules}}.
```
Comment (patrickvacek) [inline uptane-standard.md]: This PR looks good, but this sentence is slightly awkward. There are too many clauses in one long sentence. Is there a way to rewrite this to separate this out a bit better?

Comment (JustinCappos) [reply uptane-standard.md]: I've changed this, would you kindly take another look?

Comment (patrickvacek) [reply uptane-standard.md]: No problem, but it looks like you actually changed a different sentence than the one I was concerned about! I'm looking at `A primary ECU SHALL download metadata and images following the rules specified in {{TAP-5}} (if supported) or using the {{TUF-spec}}, and the metadata file renaming rules specified in {{metadata_filename_rules}}.` Sorry for the confusion!

Comment (JustinCappos) [reply uptane-standard.md]: Well, consider the other edit a bonus, then!  :)  Kindly review the latest change when you have a moment.

**Pull request merged**